test/unittests/util/test_file_utils.py

```diff
@@ -195,5 +201,14 @@ def test_create_file_in_nonexisting_dir(self):
         create_file(test_path)
         self.assertTrue(exists(test_path))
 
+    def test_get_temp_path(self):
+        temp_dir = tempfile.gettempdir()
+        temp_file = get_temp_path('example.txt')
+        self.assertEqual(temp_file, temp_dir + '/example.txt')
+        temp_path = get_temp_path('mycroft', 'audio', 'example.wav')
+        self.assertEqual(temp_path, temp_dir + '/mycroft/audio/example.wav')
+        failed_temp_path = get_temp_path(1)
```

Comment:
Maybe this could be an assertRaises check after the change to keep the exception:

```python
with self.assertRaises(TypeError):
    failed_temp_path = get_temp_path(1)
```